ui/app/reducers/app.js

```diff
@@ -354,7 +350,7 @@ function reduceApp (state, action) {
       return extend(appState, {
         currentView: {
           name: 'confTx',
-          context: 0,
+          context: action.id && indexForPending(state, action.id) || 0,
```

Comment:
Let's avoid using different conditional on the same line as it is not intuitive to read. If it has to be done, group it logically using `()`

Reply:
I changed it to use a ternary operator instead in ed9123f7.